### PR TITLE
Convert popup for file actions to transient

### PR DESCRIPTION
**Pull request by jinnovation**
Relates to #150.

### CODE REVIEW

File: kubernetes-popups.el [hunk 1]
```diff
@@ -56,16 +56,14 @@
...
-    (?c "Container" "--container=" kubernetes-utils-read-container-name))
-  :actions
-  '((?f "Find file" kubernetes-tramp-find-file)
-    (?d "Dired" kubernetes-tramp-dired))
-  :default-action 'kubernetes-tramp-dired)
+(transient-define-prefix kubernetes-file ()
+  "Work with files in Kubernetes resources."
+  [["Options"
+    ;; TODO: This doesn't currently get picked up by the suffixes
+    ("=c" "Container" "--container=" kubernetes-utils-read-container-name)]]
```
Comment (noorul) [inline kubernetes-popups.el:63]: Shouldn't we fix this before mergeing?

Comment (jinnovation) [reply kubernetes-popups.el:63]: To be clear, it does not appear that either `kubernetes-tramp-dired` or `kubernetes-tramp-find-file` pick up the container name from the popup options in `master` currently. So I would argue that we're no better or worse off than before w/ this PR, giving us the opportunity to address this TODO the Transient way after merging this PR.

Let me know your thoughts here. I'm also happy to fix this issue before switching this popup to Transient, in which case I'll open a separate PR w/ that fix.

Comment (noorul) [reply kubernetes-popups.el:63]: ok

Comment (jinnovation) [reply kubernetes-popups.el:63]: See #205 👍

**Pull request merged**